infra/deploy.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+if [[ -z ${GIT_TAG} ]]; then
+	echo "Missing git tag"
+	exit 1
+fi
+
+git checkout "${GIT_TAG}"
+gcloud container clusters get-credentials analysis-cluster --zone=us-central1-c --project=ossf-malware-analysis
+
+pushd infra/worker || echo "could not change to infra/worker directory" && exit 1
+
+echo "Were any changes made to the k8s config?"
+echo "Enter y to apply config changes and then restart workers, n to just restart, ctrl-C to exit"
+read yn
+case $yn in
+	[Yy]* )
+		echo "kubectl apply -f ."
+		kubectl apply -f .
```

Comment:
Can the files (or path) be made more explicit?

Either that, or add the output from `ls` here so we can see what is about to get applied.

Reply:
done